utils/data-migration/.eslintrc.js

```diff
@@ -19,6 +19,10 @@ module.exports = {
     'implicit-arrow-linebreak': 0,
     'comma-dangle': 'off',
     'no-loop-func': 'off',
+    'import/no-unresolved': 0,
+    'no-await-in-loop': 0,
```

Comment:
Now that these rules are disabled here, we should be able to remove all individual instances where it's disabled in this service. For example line 167 in `migrate-deals.js`.

I think there are quite a few places we can make similar changes.

Reply:
Have removed individual instances.